src/components/SearchCard.js

```diff
@@ -0,0 +1,215 @@
+import throttle from "lodash/throttle";
+import React, { useMemo, useState } from "react";
+import { constants } from "../util/helpers";
+const { Input, Card, Button, Result, Select } = require("antd");
+const { Option } = Select;
+const defaultNetwork = constants.CONTRACT_NETWORKS[0];
+
+export const SearchCard = ({ session }) => {
+  const [loading, setLoading] = useState(false);
+  const [installing, setInstalling] = useState(false);
+  const [success, setSuccess] = useState(false);
+  const [contract, setContract] = useState("");
+  const [error, setError] = useState("");
+  const [defs, setDefs] = useState([]);
+  const [network, setNetwork] = useState(defaultNetwork.value);
+
+  const resetState = () => {
+    setLoading(false);
+    setSuccess(false);
+    setInstalling(false);
+    setContract("");
+    setDefs([]);
+  };
+
+  function onChange(value) {
+    setNetwork(value);
+  }
+
+  // TEMPORARY FUNCTION TO REMOVE FUNCTIONS WITH ZERO LENGTH PARAM NAMES
+  // SEE: https://github.com/GridPlus/gridplus-sdk/issues/1540xfDcC959b0AA82E288E4154cB1C770C6c4e958a91
+  function TMP_REMOVE_ZERO_LEN_PARAMS(defs) {
```

Comment:
You shouldn't need to do this. The linked issue has been resolved. Were you having issues without this function?